src/components/CardImage/styled.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const StyledCardImage = styled.img`
```

Comment:
please have a look at my comment regarding the `<Image />` Component of next js. ✌️